applications/ParticleMechanicsApplication/custom_elements/updated_lagrangian.cpp

```diff
@@ -1699,7 +1699,8 @@ int  UpdatedLagrangian::Check( const ProcessInfo& rCurrentProcessInfo ) const
 {
     KRATOS_TRY
 
-    Element::Check(rCurrentProcessInfo);
+    KRATOS_ERROR_IF(this->Id() < 1) << "Element found with Id " << this->Id() << std::endl;
```

Comment:
I reckon this can be debug? I think it shouldn't be called everytime?!

Reply:
The base element check has this, so I guess it should be maintained as it is

Reply:
:octocat: